ui/js/directives/treeherder/log_viewer_steps.js

```diff
@@ -27,7 +27,7 @@ treeherder.directive('lvLogSteps', ['$timeout', '$q', function ($timeout, $q) {
                     $timeout(function () {
                         var raw = $('.lv-log-container')[0];
                         var line = $('.lv-log-line[line="' + linenumber + '"]');
-                        raw.scrollTop += line.offset().top - $('.run-data').outerHeight() - 15 ;
+                        raw.scrollTop += line.offset().top - $('.run-data').outerHeight() - 50 ;
```

Comment:
I plan to remove the single char whitespace `50 ;` and also below on L69, before merge.